package.json

```
@@ -27,7 +28,7 @@
   },
   "homepage": "https://github.com/axelarnetwork/axelar-cgp-solidity#readme",
   "dependencies": {
-    "@axelar-network/axelar-gmp-sdk-solidity": "^4.0.0"
+    "@axelar-network/axelar-gmp-sdk-solidity": "^4.0.1"
```

```suggestion
    "@axelar-network/axelar-gmp-sdk-solidity": "^4.0.2"
```
this can be updated again